indigo_app/templates/indigo_api/work_popup.html

```diff
@@ -1,13 +1,17 @@
 <h6>{{ work.title }}</h6>
 <p class='text-muted mb-1'>{{ work.frbr_uri }}</p>
 
-<p class='mb-1'>Published
-  {% if publication_name %} in {{ work.publication_name }} {% endif %}
-  on {{ work.publication_date|date:"Y-m-d" }}
-</p>
 {% if work.assent_date %}
   <p class='mb-1'>Assented to on {{ work.assent_date|date:"Y-m-d" }}</p>
 {% endif %}
+<p class='mb-1'>Published
+  on {{ work.publication_date|date:"Y-m-d" }}
+  {% if work.publication_name %}<br>in {{ work.publication_name }}{% endif %}
```

Comment:
I'm okay if this is all one line, it's a sentence after all (and will probably wrap in any case).